### PR TITLE
feat(AAiT-web_g3): add header component

### PR DESCRIPTION
**Pull request by surafel58**
- I added a header component which contains the navabr, the logo, and login, donate button. 
- I have defined a type for the contents of the navbar in the types folder.
Please, review the code.

image

### CODE REVIEW

File: aait/web/group-3/starter_project_group3/app/globals.css [hunk 1]
```diff
@@ -1,4 +1,5 @@
 @import url("https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap");
+
 @tailwind base;
 @tailwind components;
-@tailwind utilities;
+@tailwind utilities;
```
Comment (kidus101) [inline aait/web/group-3/starter_project_group3/app/globals.css:5]: imported @tailwind utilities; 2 times.

**Review by kidus101 — COMMENTED** [review]
Correct the PR name by following the guideline --- This is the convention for naming braches .
  
  Aait.web.g3.surafel sentayehu.header #80 
  You can see the PR naming structure below.
  
  PR Naming Convention: feat(AAiT-web_g3): add footer page

**Comment by surafel58** [comment]
> Correct the PR name by following the guideline --- This is the convention for naming braches .
> 
> Aait.web.g3.surafel sentayehu.header #80 You can see the PR naming structure below.
> 
> PR Naming Convention: feat(AAiT-web_g3): add footer page

File: aait/web/group-3/starter_project_group3/components/header/NavBar.tsx
Comment (BrookW-83) [inline aait/web/group-3/starter_project_group3/components/header/NavBar.tsx:1]: - The responsiveness for the nav bar, should be a drop down to ge the rout links , for mobile view use a three bar drop down to show the routes

**Review by kidus101 — DISMISSED** [review]

File: aait/web/group-3/starter_project_group3/components/header/NavBar.tsx
Comment (surafel58) [reply aait/web/group-3/starter_project_group3/components/header/NavBar.tsx:1]: Okay, will do that.

File: aait/web/group-3/starter_project_group3/components/header/NavLink.tsx
Comment (BrookW-83) [inline aait/web/group-3/starter_project_group3/components/header/NavLink.tsx:1]: - The "use client" is unused so remove it

File: aait/web/group-3/starter_project_group3/components/header/NavBar.tsx
Comment (BrookW-83) [reply aait/web/group-3/starter_project_group3/components/header/NavBar.tsx:1]: - In the classname, your using "flex" but not using flex properties

File: aait/web/group-3/starter_project_group3/app/page.tsx [hunk 2]
```diff
@@ -1,113 +1,6 @@
...
-              -&gt;
-            </span>
-          </h2>
-          <p className={`m-0 max-w-[30ch] text-sm opacity-50`}>
-            Instantly deploy your Next.js site to a shareable URL with Vercel.
-          </p>
-        </a>
-      </div>
-    </main>
+    <div></div>
```
Comment (BrookW-83) [inline aait/web/group-3/starter_project_group3/app/page.tsx:4]: - This divi is unused remove it

File: aait/web/group-3/starter_project_group3/components/header/NavLink.tsx
Comment (surafel58) [reply aait/web/group-3/starter_project_group3/components/header/NavLink.tsx:1]: Okay.

File: aait/web/group-3/starter_project_group3/app/page.tsx (hunk 2 above)
Comment (surafel58) [reply aait/web/group-3/starter_project_group3/app/page.tsx:4]: It is just a place holder thats why, I will add some contents when I finish with the home page.

**Pull request closed without merging**